src/pyscheduling/Problem.py

```diff
@@ -37,6 +42,200 @@ def __lt__(self, other):
         """
         return self.name < other.name
 
+class Constraints():
+
+    @staticmethod
+    def sorting_func(constraint):
+        if constraint is Constraints.W : return 0
```

Comment:
Maybe a dict that belongs to the Constraints class as a static attrbiute and maps each constraint to a value would be better ? 
This is not performance critical though